src/SolutionTemplate/UnoLibraryTemplate.netcore/CrossTargetedLibrary.csproj

```diff
@@ -1,23 +1,30 @@
 <Project Sdk="MSBuild.Sdk.Extras/3.0.23">
```

Comment:
Changing this to `Microsoft.NET.Sdk` breaks assets inclusion in package.

I think this comes from https://github.com/novotnyllc/MSBuildSdkExtras/blob/b58e1d25b530e02ce4d1b937ccf99082019cdc47/Source/MSBuild.Sdk.Extras/Build/Workarounds.targets#L89-L97

We have three options:

1. Keep using MSBuild.Sdk.Extras (I'll likely do that in this PR for simplicity)
2. Include https://github.com/novotnyllc/MSBuildSdkExtras/blob/b58e1d25b530e02ce4d1b937ccf99082019cdc47/Source/MSBuild.Sdk.Extras/Build/Workarounds.targets#L89-L97 in the template and move to Microsoft.NET.Sdk
3. Include https://github.com/novotnyllc/MSBuildSdkExtras/blob/b58e1d25b530e02ce4d1b937ccf99082019cdc47/Source/MSBuild.Sdk.Extras/Build/Workarounds.targets#L89-L97 in Uno packages. The downside is that if a user removed Uno packages from Windows head, things will break.

Reply:
I have not seen this particular error in standard projects ([this project](https://github.com/unoplatform/uno/blob/16becd50feb48f3e8ee846746a00aee1bd9fce70/src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Views/UnoQuickStart.csproj#L1) is not using extras and assets are seemingly published properly). How did you determine this workaround is needed?

Reply:
@jeromelaban The test explicitly sets `IncludeContentInPack` to false. I guess that's why it works for the project you are referring to but broke for the test.

Reply:
@Youssef1313 Should be fixed by https://github.com/unoplatform/uno/blob/7cb8e85d80c9b889e1bc3fba3d1740dca32a2768/build/uno.winui.winappsdk.targets#L92, can you try again?